object-search/flow-index.yml

```diff
@@ -0,0 +1,37 @@
+!Flow
+with:
+  logserver: false
+pods:
+  normalizer:
+    uses: pods/craft-normalize.yml
+    read_only: true
+  object:
+    uses: pods/craft-object.yml
+    read_only: true
+  resize:
```

Comment:
ok, complex situation, to keep quality of the image indexed.
Right now the `chunk.gz` contains images resized with a loss of quality because they need to be resized to be encoded. 

Steps:

1. Need to split vec.yml into a `NumpyIndexer` and `BinaryPbIndexer` in different Pods.
2. The new BinaryPbIndexer needs to receive directly the output from `craft-object` Pod.